### PR TITLE
Javascript Ex 1

### PR DESCRIPTION
**Pull request by luciapuccini**

### CODE REVIEW

File: 01-JS-DOM-APIs/resources/css/style.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+section {
+  text-align: center;
+  visibility: hidden;
+}
+#slideSource {
```
Comment (troyax) [inline 01-JS-DOM-APIs/resources/css/style.css:5]: selectors should be writting in other way as we reviewed in the topic 0. For this case it should be `#slide-source`

File: 01-JS-DOM-APIs/resources/js/main.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  callEvents();
+};
+document.getElementById('alert-btn').onclick =function()
+{
+alert('this is an alert! o_o');
+};
+//fetching data
+document.getElementById('joke').addEventListener
+  ('click', fGetText);
+   function fGetText()
```
Comment (troyax) [inline 01-JS-DOM-APIs/resources/js/main.js]: All this function should not be tabbed

File: 01-JS-DOM-APIs/resources/js/main.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  // but for clarity
+  callEvents();
+};
+document.getElementById('alert-btn').onclick =function()
+{
+alert('this is an alert! o_o');
+};
+//fetching data
+document.getElementById('joke').addEventListener
+  ('click', fGetText);
```
Comment (troyax) [inline 01-JS-DOM-APIs/resources/js/main.js]: I'd rather write this in line 21 too

**Review by troyax — CHANGES_REQUESTED** [review]

**Review by troyax — APPROVED** [review]

**Comment by luciapuccini** [comment]
End Topic 1 JS

**Pull request merged**